ghedesigner/manager.py

```diff
@@ -860,8 +860,8 @@ def run_manager_from_cli_worker(input_file_path: Path, output_directory: Path) -
 
 
 @click.command(name="GHEDesignerCommandLine")
-@click.argument("input-path", type=click.Path(exists=True))
-@click.argument("output-directory", type=click.Path(exists=True), required=False)
+@click.argument("input-path", type=click.Path(exists=True), required=True)
+@click.argument("output-directory", type=click.Path(exists=False), required=False)
```

Comment:
OK, so it guarantees the output directory should **not** exist, that seems quite reasonable.